### DISCUSSION TITLE
How to download multiple link, what is the delimiter for multiple link

### DISCUSSION THREAD

**Discussion by thiendt2k1** [anasty17/mirror-leech-telegram-bot#1741] [Q&A]
I am currently trying to leech multiple links, for specific 50 ggdrive link, copy from ggdrive, but when i paste in to telegram, it is too long, and i cant paste all, but even when i paste only 10, it still only leech the first file, not the other 9 links. 

**Example links:**
```
https://drive.google.com/file/d/1vFo-apm9yackQmvzBEy78ucgghqRoC_h/view?usp=drive_link, 
https://drive.google.com/file/d/1av-GkzCrT0n6HW5E_HEz0yMr6lLOu05V/view?usp=drive_link, https://drive.google.com/file/d/1p5HomqmRz914npnfBoNOUM4hIfIqFMHU/view?usp=drive_link
``` 
**Command i used:**
```python
/leechz -m 10 -sd Folder1 #reply to the 10 links
``` 
Then i tried to put them all into a txt file with each link a line, still not work :)

So can anyone tell me how to use leech/mirror multiple link correctly? TIA

**Comment by anasty17** [comment]
Where did u get those args !?
